zb-constants.js

```diff
@@ -145,6 +146,8 @@ const CLUSTER_ID = {
   RELATIVE_HUMIDITY: zclId.cluster('msRelativeHumidity').value,
   SEMETERING: zclId.cluster('seMetering').value,
   SSIASZONE: zclId.cluster('ssIasZone').value,
+  SSIASACE: zclId.cluster('ssIasAce').value,
```

Comment:
What are these 2 new cluster IDs for? They're not used within the other changes.

Reply:
I remove them, I will make other tests later